test/models/device.test.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const device = require('../../models/Device');
+
+
+describe('When given a name and id', () => {
+  it('returns object with name, id, and _id', () => {
+    const result = device({ name: 'test-name', id: 'test-id' });
+    expect(result.name).toBe('test-name');
+    expect(result.id).toBe('test-id');
+    expect(result._id).toEqual(expect.stringMatching(/[0-9a-f]{24}/g));
```

Comment:
because this is an implementation detail of mongoose and not part of your own business logic, I would leave it out of your tests. you have your own device id that you have generated for your own purposes.

Reply:
done